components/banner/index.js

```diff
@@ -0,0 +1,60 @@
+import Router from 'next/router'
+import TokenSaleDetails from './tokenSaleDetails';
+import Countdown from './countdown';
+import stylesheet from './banner.scss'
+import { getSecondsUntilNextPeriod } from '../constants'
+
+class Banner extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      active: this.props.currentDayServer ? 2 : 1,
+    };
+  }
+
+  componentWillReceiveProps(nextProps){
+    if((nextProps.currentDay > 0 || nextProps.currentDayServer > 0) && this.state.active === 1){
+      this.setState({
+        active: 2,
```

Comment:
Same, I don't know what's going on here. If possible, please rename these numbers with semantic variables that can provide some context.